### PR TITLE
Add possibility to use custom CSVFormat in expression processor

### PR DESCRIPTION
**Pull request by vkepin**
Configurable with property in Spring expression way:
expression.csv-to-parquet.csv-format=T(org.apache.commons.csv.CSVFormat).DEFAULT

### CODE REVIEW

File: vividus-plugin-parquet/src/main/java/org/vividus/bdd/expression/ConvertCsvToParquetFileExpressionProcessor.java [hunk 1]
```diff
@@ -90,4 +90,9 @@ private void write(File file, String avroSchemaPath, List<Map<String, String>> d
             }
         }
     }
+
+    public void setCsvReader(CsvReader csvReader)
```
Comment (valfirst) [inline vividus-plugin-parquet/src/main/java/org/vividus/bdd/expression/ConvertCsvToParquetFileExpressionProcessor.java]: use constructor

File: vividus-plugin-parquet/src/main/resources/properties/defaults/run.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+expression.csv-to-parquet.csv-format=T(org.apache.commons.csv.CSVFormat).DEFAULT
```
Comment (valfirst) [inline vividus-plugin-parquet/src/main/resources/properties/defaults/run.properties]: it doesn't look really user-friendly, any ideas how to simplify `T(org.apache.commons.csv.CSVFormat).DEFAULT` -> `DEFAULT`

File: vividus-plugin-parquet/src/main/java/org/vividus/bdd/expression/ConvertCsvToParquetFileExpressionProcessor.java [hunk 3]
```diff
@@ -45,7 +45,12 @@
     private static final int CSV_PATH_GROUP = 1;
     private static final int SCHEMA_PATH_GROUP = 2;
 
-    private final CsvReader csvReader = new CsvReader();
+    private CsvReader csvReader;
```
Comment (uarlouski) [inline vividus-plugin-parquet/src/main/java/org/vividus/bdd/expression/ConvertCsvToParquetFileExpressionProcessor.java]: final?

File: vividus-plugin-parquet/src/test/java/org/vividus/bdd/expression/ConvertCsvToParquetFileExpressionProcessorTests.java [hunk 4]
```diff
@@ -35,11 +35,12 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
+import org.vividus.csv.CsvReader;
 
 class ConvertCsvToParquetFileExpressionProcessorTests
 {
-    private final ConvertCsvToParquetFileExpressionProcessor processor =
-            new ConvertCsvToParquetFileExpressionProcessor();
+    private ConvertCsvToParquetFileExpressionProcessor processor = new ConvertCsvToParquetFileExpressionProcessor(
```
Comment (uarlouski) [inline vividus-plugin-parquet/src/test/java/org/vividus/bdd/expression/ConvertCsvToParquetFileExpressionProcessorTests.java]: what is the reason you removed final?

**Pull request merged**